docs/guide/structure-applications.md

```diff
@@ -330,15 +330,15 @@ image size as a parameter like the following:
 ]
 ```
 
-Then in your code where you need to use the size value, you can simply use the code like the following:
+Then in your code where you need to use the size value, you can simply use code like the following:
```

Comment:
I think in this case it is fine because it reads "use code like the following". In the other places it was a separated statement. Thanks for noting though.